contrib/pylibpod/pylibpod/libs/containers.py

```diff
@@ -0,0 +1,207 @@
+"""Models for manipulating containers and storage."""
+import collections
+import functools
+import json
+
+
+class Container(collections.UserDict):
+    """Model for a container."""
+
+    def __init__(self, client, id, data):
+        """Construct Container Model."""
+        super(Container, self).__init__(data)
+
+        self._client = client
+        self._id = id
+
+        self._refresh(data)
+        assert self._id == self.data['id'],\
```

Comment:
If data contains an `id` key, why bother with `_id` and `__init__`'s separate `id` argument/